src/navigate/Navigator.js

```diff
@@ -112,6 +112,12 @@ define([
             // modelview matrix and before computing the near clip distance. The far clip distance depends on the
             // modelview matrix, and the near clip distance depends on the far clip distance.
             this.farDistance = WWMath.horizonDistanceForGlobeRadius(globeRadius, eyePos.altitude);
+
```

Comment:
We discussed the NASA team handling the changes necessary to support a far clip distance that doesn't obscure the horizon. Adding a large constant here will cause more shapes and tiles to be considered each frame, especially when the viewer is close to the ground. We'll need to find another way to handle this change.

One possibility is for the ESA team to implement the far clip convention found in WWAndroid's source in WorldWindow.computeViewingTransform, and test the resultant performance difference. In particular, we need to quantify the cost of increasing the far clip distance on terrain tile count and image tile count.